lib/couchbase-orm/json_schema.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require "json"
+require 'json-schema'
+require 'singleton'
+
+module CouchbaseOrm
+
+  module JsonSchema
+
+    class Loader
+      include Singleton
+
```

Comment:
This configuration is important and must be highlighted with a class constant... maybe, if needed by another class, it should even be declared a step higher, right in `CouchbaseOrm::JsonSchema` 

```suggestion
      SCHEMAS_DIRECTORY = ENV['CB_ORM_JSON_SCHEMA_PATH']

```